pages/docs/visualizing-data/period-comparisons.mdx

```diff
@@ -0,0 +1,18 @@
+import { Callout } from "nextra-theme-docs";
+
+
+# Period-over-period Comparisons
+
+Period-over-period comparisons allow you to compare timeseries data from different periods of time. They can be configured via the control panel on the right-hand side of the Data Explorer, under the `Compare` heading.
```

Comment:
Maybe worth mentioning that this is all happening as part of the underlying SQL query, which is reflected in the SQL query you see in the SQL runner?